packages/application-shell/package.json

```diff
@@ -18,11 +18,21 @@
   "publishConfig": {
     "access": "public"
   },
-  "main": "./src/index.js",
-  "module": "./src/index.js",
-  "engines": {
-    "node": ">=8",
-    "npm": ">=5"
+  "main": "dist/application-shell-index.cjs.js",
+  "module": "dist/application-shell-index.es.js",
+  "files": [
+    "dist",
+    "package.json",
+    "LICENSE",
+    "README.md"
+  ],
+  "scripts": {
+    "prebuild": "rimraf dist/**",
+    "build": "npm run build:es && npm run build:cjs",
+    "build:cjs": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f cjs ./index.js --dir ./dist --chunkFileNames application-shell-[name]-[hash].cjs.js --entryFileNames application-shell-[name].cjs.js --experimentalCodeSplitting",
+    "build:es": "cross-env NODE_ENV=production rollup -c ../../rollup.config.js  -f es ./index.js --dir ./dist --chunkFileNames application-shell-[name]-[hash].es.js --entryFileNames application-shell-[name].es.js --experimentalCodeSplitting",
+    "build:es:watch": "npm run build:es -- -w",
+    "test:build:es": "cross-env NODE_ENV=development rollup -c ../../rollup.config.js  -f es ./src/test-utils.js --o ./dist/test-utils/index.js"
```

Comment:
I would add this command to `yarn build`  as well, so that the full package gets built with `yarn build`.